rpcs3/Emu/SysCalls/Modules/sys_mempool.cpp

```diff
@@ -1,40 +1,129 @@
 #include "stdafx.h"
 #include "Emu/Memory/Memory.h"
 #include "Emu/System.h"
+#include "Emu/IdManager.h"
 #include "Emu/SysCalls/Modules.h"
 
 #include "sysPrxForUser.h"
 
 extern Module sysPrxForUser;
 
+using sys_mempool_t = u32;
+
+struct memory_pool_t
```

Comment:
@tambry shouldn't `u64 chunk_size;` and others be be_t ? I was wondering because it seems all the other structs in sys_ have be_t.